tools/run_tests/run_tests.py

```diff
@@ -727,7 +727,10 @@ def _start_port_server(port_server_port):
     while True:
       if waits > 10:
         port_server.kill()
+      if port_server.poll() is not None:
         print "port_server failed to start"
+        port_log = open('portlog.txt', 'r').read()
+        print port_log
```

Comment:
How about 
```
with open('portlog.txt, 'r') as f:
  for line in f:
    print line,
```
that works even if the log is huge and also takes care of closing the file.

(the trailing comma prevents `print` from printing the default \n after each line)